CMake/cdat_modules_extra/runtest.in

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 echo "ACTIVATING ENV:"@CONDA_ENVIRONMENT_NAME@
-source activate @CONDA_ENVIRONMENT_NAME@
+while [[ `which python` != *"@CONDA_ENVIRONMENT_NAME@"* ]]
+do
+    source activate @CONDA_ENVIRONMENT_NAME@
+done
```

Comment:
@danlipsa because sometimes the environement does not get activated. I suspect some concurency issues within conda, too many process trying to activate at same time, so I keep trying until it's actually activated. We could/should probably improve the test to see if it's activated (using `conda env list` or something like that) but that would be another PR.